rockpaperscissors/src/modules/helpers.js

```diff
@@ -0,0 +1,19 @@
+import {CHOICES} from "./consts";
+import { NAMES } from "./consts";
+
+
+export function renderChoice(){
+    return CHOICES[Math.floor(Math.random() * CHOICES.length)]
+};
+
+export function isRock(choice){
+    if(choice === NAMES[0]) return NAMES[0];
+};
```

Comment:
the you can use value from consts and don't bother about element order in array, also if its name id `isXXX` should return boolean 

```suggestion
export function isRock(choice){
    return choice === NAME_KEY.ROCK;
};
```